packages/client/ui/react-ui/src/providers/auth/AuthFormProvider.tsx

```diff
@@ -47,6 +49,7 @@ export const AuthFormProvider = ({
 }: { children: ReactNode; initialState: ContextInitialStateProps }) => {
     const { crossmintAuth } = useCrossmintAuth();
     const [step, setStep] = useState<AuthStep>("initial");
```

Comment:
I'm thinking since we don't reset the error when they navigate to another connector in example: 
1. user enters invalid email, the error display at the top nicely
2. user clicks on farcaster login, error still displays from invalid email.

one way I can think to fix this would be to add a useEffect inside this provider here

```
useEffect(() => {
  if(error != null){
    setError(null)
  }
}, [step, error])
```
it's possible i'm missing something in that snippet code but we might want to fix this somehow. 

![Screenshot 2024-11-12 at 11 01 24 AM](https://github.com/user-attachments/assets/fbd10061-b163-4b8d-9564-a0bfc6e4ab6a)

Reply:
That useEffect is proving to be a bit more problematic. What I'm going to do is so that on all clicks to the `<AuthFormBackButton>` it'll reset the error

Reply:
You could do it that way too, you'll also have to reset the error when clicking on any provider

Reply:
yup, did that too. more stable that way

Reply:
for sure, i think the explicit approach will be best to prevent future bugs too 👍

Reply:
hmm, it still appears to be broken on embedded. 
![Screenshot 2024-11-12 at 4 18 43 PM](https://github.com/user-attachments/assets/5670347f-7e78-456e-b2c4-18f5f21344e5)